terraform/ecs.tf

```diff
@@ -0,0 +1,98 @@
+resource "aws_iam_role" "ecs_task_execution_role" {
+  name               = "ecs_task_execution_role"
+  assume_role_policy = "${data.aws_iam_policy_document.assume_role_policy.json}"
+}
+
+data "aws_iam_policy_document" "assume_role_policy" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role_policy_attachment" "ecs_task_execution_role_policy" {
+  role       = "${aws_iam_role.ecs_task_execution_role.name}"
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
+}
+
+resource "aws_ecs_service" "my_app_service" {
+  name            = "my_app_service"     # Name the service
+  cluster         = "${aws_ecs_cluster.my_app_cluster.id}"   # Reference the created Cluster
+  task_definition = "${aws_ecs_task_definition.my_app_task.arn}" # Reference the task that the service will spin up
+  launch_type     = "FARGATE"
+  desired_count   = 2 # Set up the number of containers to 2
+
+  load_balancer {
+    target_group_arn = "${aws_lb_target_group.target_group.arn}" # Reference the target group
+    container_name   = "${aws_ecs_task_definition.my_app_task.family}"
+    container_port   = 3000 # Specify the container port
+  }
+
+  network_configuration {
+    subnets          = ["${aws_default_subnet.default_subnet_a.id}", "${aws_default_subnet.default_subnet_b.id}"]
+    assign_public_ip = true     # Provide the containers with public IPs
+    security_groups  = ["${aws_security_group.service_security_group.id}"] # Set up the security group
+  }
+}
+
+resource "aws_ecs_task_definition" "my_app_task" {
+  family                   = "my_app_task" # Name your task
+  container_definitions    = <<DEFINITION
+  [
+    {
+      "name": "my_app_task",
+      "image": "${aws_ecr_repository.my_app_repo.repository_url}",
+      "essential": true,
+      "logConfiguration":{
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "${aws_cloudwatch_log_group.my_app.name}",
+          "awslogs-region": "us-east-1",
+          "awslogs-stream-prefix": "web"
+        }
+      },
+      "environment": [
+        {
+          "name": "DATABASE_HOST",
+          "value": "${split(":", aws_db_instance.rds.endpoint)[0]}"
```

Comment:
```suggestion
          "value": "${aws_db_instance.rds.address}"
```

Reply:
`address` is just the hostname without the port.

I could be wrong, because all these things are named badly and I often have to try a few before I get what I actually want.